src/esp/nav/PathFinder.cpp

```diff
@@ -34,6 +35,35 @@ namespace Cr = Corrade;
 namespace esp {
 namespace nav {
 
+struct MultiGoalShortestPath::Impl {
+  Impl() = default;
+  ~Impl() = default;
```

Comment:
These aren't needed either I think ([rule of zero](https://en.cppreference.com/w/cpp/language/rule_of_three)).